core/3d/Mesh.h

```diff
@@ -322,6 +322,10 @@ class AX_DLL Mesh : public Object
     std::vector<float> _spotLightUniformRangeInverseValues;
 
     std::string _texFile;
+
+    // add by binxiaojiao
+public:
+    std::function<void()>& GetVisibleChanged() { return _visibleChanged; }
```

Comment:
This should follow the convention used in Axmol, which is to use a lowercase first character, so it should be `getVisibleChanged()`

Reply:
I prefer public callback like engine callback renderer comnand